web-frontend/src/layouts/WalletLayout/WalletLayout.tsx

```diff
@@ -62,42 +62,42 @@ function WalletLayout(props: WalletProps) {
       icon: <FiHome />,
       active: isActive(MenuItem.Home),
       footer: false,
-      onClickItem: () => navigate(routeFor(MenuItem.Home)),
+      onClickItem: () => navigate(routeFor(MenuItem.Home), { state }),
```

Comment:
Curiosity: What's is that `{ state }`

Reply:
we need to pass the same state so that the WalletLayout have it

Reply:
@mazmassa Because when you try to go on setting page, page want to have nickname. And nickname is present on state.

Reply:
@mazmassa We can optimize this in another PR. This should have been done before.

Reply:
Ok! As you wish!

Reply:
Ok! As you wish! I am afraid we forget...  :/